### PR TITLE
Add 0.66 MOM6 set up.

### PR DESCRIPTION
**Pull request by yvikhlya**
Add 0.66 MOM6 set up.

### CODE REVIEW

**Comment by sanAkel** [comment]
@yvikhlya:
- I added`global` contingency from https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/712#issuecomment-1453635355 
- Please provide `MAPL_Tripolar.cdl` in `mom6-0.66deg/MOM6_GEOSPlug/mom6_app/` that you have generated for _this_ resolution.
- Add [organization information](https://github.com/GEOS-ESM/MAPL/blob/ecb6119776671432a3952575547ad2752d19544e/base/ESMFL_Mod.F90#L1123) to https://github.com/GEOS-ESM/GEOS_OceanGridComp/blob/0c394ff1e65210cd33a9942051e6839ce54c76f5/MOM6_GEOSPlug/mom6_app/mkMAPLTripolar/MAPL_Tripolar.cdl#L66-L68

**Comment by yvikhlya** [comment]
I was actually trying to avoid polluting repo with multiple copies of MAPL_Tripolar.cdl as unnecessary. This is a trivial config edit for each grid, which is needed anyway, because location of input data is not permanent. Anyway, if it helps, here it is.

This PR can be merged independently from other PRs, it will not break anything.

**Comment by sanAkel** [comment]
> I was actually trying to avoid polluting repo with multiple copies of MAPL_Tripolar.cdl as unnecessary. This is a trivial config edit for each grid, which is needed anyway, because location of input data is not permanent. Anyway, if it helps, here it is.
> 

Thanks for adding it. Having it allows quick reproducibility, or else everyone has to make it. Also good to have the `metadata`
 recorded here.

> This PR can be merged independently from other PRs, it will not break anything.

Yes, true.

File: MOM6_GEOSPlug/mom6_app/540x458/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is 540x458x50 configuration (0.66 degree) from NCEP(?)
```
[the remark this answers is not in the document]
Comment (sanAkel) [reply MOM6_GEOSPlug/mom6_app/540x458/README.md:1]: @yvikhlya have you pushed this update?

**Review by sanAkel — CHANGES_REQUESTED** [review]

**Pull request merged**